src/vivarium/framework/values/pipeline.py

```diff
@@ -65,7 +65,7 @@ def __call__(self, population_mgr: PopulationManager, *args: Any, **kwargs: Any)
         else:
             raise TypeError(
                 "The source of an attribute pipeline must be a callable or a list "
-                f"of column names, but got {type(self._source)}."
+                f"of source column names, but got {type(self._source)}."
```

Comment:
This is fine for now, but we should make a note of all references to private/source columns so we can standardize our naming before we're done.